extensions/wikia/Email/templates/icon.mustache

```diff
@@ -1,3 +1,3 @@
-{{# iconLink}}<a href="{{iconLink}}">{{/iconLink}}
+{{# iconUrl}}<a href="{{iconUrl}}">{{/iconUrl}}
```

Comment:
I think `iconLink` is used in `FounderController` class. Maybe it would be safer to stick with `iconLink`?